### ISSUE TITLE
NVDA not reading values from HTML 5 input type="number" field

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#12752]
### Steps to reproduce:
Please find the attached file with this issue and do the following:
- Find the "Contact number" field in the attached file.
- focus the field using tab key.
- enter any numeric value in it.
- move your focus away from the field and refocus it.
### Actual behavior:
NVDA doesn't read the entered value.
Also, when up/down arrow keys are pressed in input type=number field, the value increases/decreases by 1 respectively.
The same is also not announced by the NVDA.
### Expected behavior:
NVDA should announce the entered value as well as the value changed using up/down arrow keys.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA 2021.1
#### Windows version:
Windows 21 H1
#### Name and version of other software in use when reproducing the issue:
Google chrome Version 92.0.4515.159 (Official Build) (64-bit)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not able to recall in which version it was working, but I have seen it working.
I did try NVDA 2020.4, it is not working there as well!
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
### Attachment file
Please remove .txt extention from the file
[NVDA_HTML5_number_test.html.txt](https://github.com/nvaccess/nvda/files/7015002/NVDA_HTML5_number_test.html.txt)

**Comment by seanbudd** [comment]
I believe this is a duplicate of #5422
